packages/manager/modules/office/src/office.module.js

```diff
@@ -2,19 +2,14 @@ import angular from 'angular';
 import 'angular-translate';
 import '@ovh-ux/ui-kit';
 import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
-import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
-
 import routing from './office.routing';
+import component from './office.component';
 
 const moduleName = 'ovhManagerOfficeLicenses';
 
 angular
-  .module(moduleName, [
-    ngTranslateAsyncLoader,
-    'oui',
-    'pascalprecht.translate',
-    ListLayoutHelper.moduleName,
-  ])
+  .module(moduleName, [ngTranslateAsyncLoader, 'oui', 'pascalprecht.translate'])
+  .component('microsoftOffice', component)
```

Comment:
**nitpick:** Component name is generic, can we rename it to `microsoftOfficeLicense`?

Reply:
sure Anoop